php/Dockerfile

```diff
@@ -7,7 +7,7 @@ ENV DEBIAN_FRONTEND noninteractive
 RUN apt-get update && \
     apt-get --no-install-recommends --no-install-suggests --yes --quiet install \
         apt-transport-https bash-completion ca-certificates curl git gnupg imagemagick \
-        less mongodb-clients mysql-client perceptualdiff ssh-client sudo vim wget && \
+        less mongodb-clients mysql-client perceptualdiff ps ssh-client sudo vim wget && \
```

Comment:
`ps` is mandatory for the CI. The Jenkinsfile cannot work without it.
The SSH client is also mandatory to allow the install of pim-enterprise-standard, as it uses the host ssh key.
For vim, it is indeed very arguable.